Assets/Scenes/SampleScene.unity

```diff
@@ -176,7 +176,7 @@ Camera:
   far clip plane: 1000
   field of view: 60
   orthographic: 1
-  orthographic size: 5
+  orthographic size: 1
```

Comment:
In my opinion, the previous camera size (5) in `SampleScene` is too zoomed out, and the changed camera size (1) in that scene is too zoomed in. It's up to you whether to change the size or not because the camera size is unrelated to this pull request.